src/daemon/abrt-action-save-package-data.conf

```diff
@@ -1,28 +1,49 @@
-# With this option set to "yes",
-# only crashes in signed packages will be analyzed.
-# the list of public keys used to check the signature is
-# in the file gpg_keys
+# Configuration file for the ABRT daemon
+# --------------------------------------
+
+# If enabled, only crashes in signed packages will be analyzed. The list of
+# trusted public keys used to verify the signature is in /etc/gpg_keys.conf.
 #
-# How can I check the GPG key used to sign an installed package on
-# Red hat Enterprise Linux:
+# "How can I check the GPG key used to sign an installed package on
+# Red hat Enterprise Linux":
 # https://access.redhat.com/solutions/1120013
 #
-# Product Signing (GPG) Keys:
+# "Product Signing (GPG) Keys":
 # https://access.redhat.com/security/team/key
 #
-OpenGPGCheck = yes
+# Default: yes
+#
+# OpenGPGCheck = yes
 
-# Blacklisted packages
+# Crashes in packages listed here will be ignored by ABRT.
 #
-BlackList = nspluginwrapper, valgrind, strace, mono-core, bash
+# Example: nspluginwrapper, valgrind, strace, mono-core, bash
+# Default: (empty)
+#
+# BlackList =
 
-# Process crashes in executables which do not belong to any package?
+# Crashes in paths matching the glob patterns in this list will be ignored
+# by ABRT.
+#
+# Example: /usr/share/doc/*, */example*, /usr/bin/nspluginviewer,
+#          /usr/lib*/firefox/plugin-container
+# Default: (empty)
 #
-ProcessUnpackaged = no
+# BlackListedPaths =
 
-# Blacklisted executable paths (shell patterns)
+# If enabled, ABRT will catch crashes of all programs in the system (except
+# those suppressed by other settings). If disabled, only crashes in
+# installed packages will be handled.
```

Comment:
I feel like this is overly complicated description.

`If enabled, ABRT will process crashes from unpackaged executables.` ?